media-libs/jpeg/jpeg-9a.recipe

```diff
@@ -65,6 +65,9 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
+defineDebugInfoPackage jpeg \
+	$libDir/libjpeg.so.9.1.0
```

Comment:
Replacing "so.9.1.0" with "so...*" didn't work. Replacing with "so.*" also failed:
objcopy: '/packages/jpeg-9a-2/.self/lib/libjpeg.so.*': No such file

By the way, the version was already hardcoded in two other places. I was thinking about defining a variable libVersion but decided to leave it alone.